azure/storage/blob/models.py

```diff
@@ -219,5 +219,10 @@ class BlobSharedAccessPermissions(object):
     '''
     WRITE = 'w'
 
+    '''
+    List the contents of a container.
+    '''
+    LIST = 'l'
```

Comment:
No need to add this. The real problem is that the sample should be using ContainerSharedAccessPermissions which does contain list (look just above in the code)